Source/Core/Core/Boot/Boot.cpp

```diff
@@ -291,9 +292,9 @@ bool CBoot::Load_BS2(const std::string& boot_rom_filename)
 
 static void SetDefaultDisc()
 {
-  const SConfig& config = SConfig::GetInstance();
-  if (!config.m_strDefaultISO.empty())
-    SetDisc(DiscIO::CreateVolumeFromFilename(config.m_strDefaultISO));
+  std::string default_iso = Config::Get(Config::MAIN_DEFAULT_ISO);
```

Comment:
Can this be ``const``?

Reply:
Easy enough! Added a `const` in MainWindow too, for good measure